CMakeLists.txt

```diff
@@ -951,35 +951,18 @@ if(NOT CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_CURRENT_BINARY_DIR)
     endif()
 endif()
 
-# Refer to prefix symbolically to ease relocation by end user,
-# as Makefile-generated .pc file does.
-string(FIND "${CMAKE_INSTALL_INCLUDEDIR}" "${CMAKE_INSTALL_PREFIX}/" INCLUDEDIR_POS)
```

Comment:
The idea is to drop this complexity. If a user wants to customize the install layout while keeping things relocatable, then just use `CMAKE_INSTALL_INCLUDEDIR=myincludedir` instead of an absolute path, as suggested by the docs: https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html#result-variables